app/config/collections.php

```diff
@@ -1059,7 +1059,7 @@
                 'filters' => [],
             ],
             [
-                '$id' => 'sessions',
+                '$id' => 'sessions', // TODO: Subqueries seems to be impossible to index. We dont have '.includes()' yet
```

Comment:
yeah, that we can skip completely for now 👍🏻  only on root attributes - no nesting yet.